public/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="header-logo">
+            <i class="fas fa-dove fa-2x"></i>
+        </div>
+
+        <div class="header-link">
+            <a href="./about.html">About</a>
+        </div>
+    </div>
+
+    <div class="Title">
+        <h1>THE BLOG</h1>
+    </div>
+
+    <div class="Index-Content">
+        <div class="Blog1">
```

Comment:
All the same styles will be attached to every blog - should just have one class, like "blog-card". As well, this is probably best as an article tag. 
- [ ] change class to "blog-card" 
- [ ] make into article tag